src/pybamm/simulation.py

```diff
@@ -182,7 +182,8 @@ def set_up_and_parameterise_experiment(self, solve_kwargs=None):
         reduces simulation time since the model formulation is efficient.
         """
         parameter_values = self._parameter_values.copy()
-
+        msg = "pybamm.simulation.set_up_and_parameterise_experiment is deprecated and not meant to be accessed by users."
+        warnings(msg, DeprecationWarning)
```

Comment:
Two problems here:
1. You need to make it warnings.warn
2. You need to explicitly set stacklevel

Reply:
Like this?
<img width="704" alt="image" src="https://github.com/user-attachments/assets/92bcf02e-157c-4024-9925-ec70a2c740b9">

Reply:
Yes